CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing to ArmoniK
+
+Hi! We are excited that you are interested in ArmoniK and want to make it even better! This document is a general guide we would like our contributors to follow. Before submitting your contribution, please make sure to take a moment and read through the following guide.
+
+## Discuss First !
+
+Before attempting to make a feature pull request, open an issue and discuss it with us. It will help maintainers understand what you want to do as well as make sure that you go in a direction that is suitable for ArmoniK' future.
```

Comment:
Add a paragraph about the discussion section in ArmoniK if they have a question or an idea.